utils/build/docker/nodejs_otel/express4-otel.Dockerfile

```diff
@@ -16,6 +16,7 @@ WORKDIR /usr/app
 ENV NODE_ENV=production
 
 RUN npm install
+RUN npm install "express@^4.17.2" "apollo-server-express@^3.13.0" "express-mongo-sanitize@^2.2.0"
```

Comment:
you're using the caret ^ in this dockerfile but not the other one, is that intended ?